stable/redis-ha/templates/redis-ha-service.yaml

```diff
@@ -1,3 +1,4 @@
+{{- if not .Values.haproxy.enabled }}
```

Comment:
They don't need to be mutually exclusive. This service can still be used for access within the cluster.